Tavisca.Bootcamp.LanguageBasics.Exercise1/FixMultiplication.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tavisca.Bootcamp.LanguageBasics.Exercise1
+{
+    public class FixMultiplication
+    {
+        public static int FindDigit(string equation)
+        {
+            // a * b = c
+            //Extrct 3 numbers
+
+            var indexOfMultiply = equation.IndexOf('*');
+            var indexOfEqual = equation.IndexOf('=');
+            var multiplier1 = equation.Substring(0, indexOfMultiply);
+            var multiplier2 = equation.Substring(indexOfMultiply + 1, indexOfEqual - indexOfMultiply - 1);
+            var result = equation.Substring(indexOfEqual + 1, equation.Length - indexOfEqual - 1);
+
+            var missingDigitPlaceHolder = multiplier1.Contains('?') ? 1 : multiplier2.Contains('?') ? 2 : 3;    //finding position of '?' in the string
```

Comment:
missingDigitPlaceHolder is not a valid name here... place holder is ? and you are finding operand which contains the place holder. so you should use it like

>  var missingDigitPlaceHolder='?';
> var positionOfMissingDigitPlaceHolder = multiplier1.Contains('?') ? 1  : multiplier2.Contains('?') ? 2 : 3;